www/src/public/js/controllers/builder/main.js

```diff
@@ -144,38 +144,42 @@
         var loadCharacterLists = function() {
             var lists = [];
 
-            // load lists
+            // load lists and version check
             if ($cookies.get("cookie-consent")) {
                 var listVersion = -1;
-                var listCookieStr = localStorage.getItem("cl3")
+                var listCookieStr = localStorage.getItem("cl3");
                 
-                // version check
-                if (localStorage.getItem("cl1")) {
-                    listCookieStr = localStorage.getItem("cl1");
-                    listVersion = 1;
-                } 
-                if (localStorage.getItem("cl2")) {
-                    listCookieStr = localStorage.getItem("cl2");
-                    listVersion = 2;
-                }
-                if (localStorage.getItem("cl3")) {
-                    listCookieStr = localStorage.getItem("cl3");
+                if (listCookieStr) {
                     listVersion = 3;
-                }                    
+                }
+                else {
+                    listCookieStr = localStorage.getItem("cl2");
+                    if (listCookieStr) {
+                        listVersion = 2;
+                    }
+                    else {
+                        listCookieStr = localStorage.getItem("cl1");
+                        if (listCookieStr) {
+                            listVersion = 1;
+                        }
+                        else {
+                            listVersion = "cl";
```

Comment:
this else statement should do a `localStorage.getItem("cl")`. As far as the `listVersion` goes, both `cl1` and `cl` are considered version 1, so you can set that outside of this specific if/else statement, before you try `getItem("cl1")`. This part is really weird and dated so I don't blame you for the confusion

Reply:
Fixed!

Reply:
This is still an issue because you're setting the listVersion to "cl", when it should be set to 1. You can just move the `listVersion = 1` out of the if/else since both the if and else are going to set it to 1, here.